docs/content/hooks-and-utils/date/date.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: DateFormatter
```

Comment:
Not about "formatter", maybe "DateValue"? Or "DateFormat"?

Reply:
I would say DateFormat is a good name